src/FlipMove.js

```diff
@@ -69,7 +69,7 @@ class FlipMove extends Component {
     // can complete. Because we cannot mutate props, we make `state` the source
     // of truth.
     this.state = {
-      children: props.children.map(child => ({
+      children: Children.toArray(props.children).map(child => ({
```

Comment:
Since `prop-converter` handles converting `children` to array, is it necessary in this file?

The goal of `prop-converter` is to abstract away all the tedious business of stuff like this, so if we can isolate it to that HOC I'd be happy :)

That said, I don't have much experiencing aliasing React-like libraries, so let me know if there's a reason that this isn't doable.